backend/entities/LookupEntity.cs

```diff
@@ -29,6 +31,7 @@ public abstract class LookupEntity : BaseEntity
         /// get/set - The sort order of the lookup item.
         /// </summary>
         /// <value></value>
+        [NotMapped]
```

Comment:
Why is this not mapped?

Reply:
I'll add this when I regenerate the migration.